plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -138,6 +139,15 @@ func findNodesThatFit(pod *api.Pod, podLister algorithm.PodLister, predicateFunc
 			filtered = append(filtered, node)
 		}
 	}
+	if len(filtered) > 0 && len(extenders) != 0 {
+		for _, extender := range extenders {
+			filteredList, err := extender.Filter(pod, &api.NodeList{Items: filtered})
```

Comment:
should you also pass the full machinetoPods map, since predicate() on L121 gets it (for the node on the current iteration)?